Tests/FluentAssertions.Equivalency.Specs/CollectionSpecs.cs

```diff
@@ -311,21 +310,27 @@ public void When_collection_of_same_count_does_not_match_it_should_include_at_mo
             .Message.Should().Contain("[9]").And.NotContain("[10]");
     }
 
+#if NETFRAMEWORK || NET60_OR_GREATER
```

Comment:
This is always true when running on net47 or net6, so can be removed.

Note: If we had kept this, `NET60_OR_GREATER` should have been `NET6_0_OR_GREATER`. This is the reason for the reduced CC.
We're not collecting CC for net47 - I'm working on that right now.